amp/WEB-INF/src/org/digijava/module/aim/form/TeamAuditForm.java

```diff
@@ -21,34 +23,11 @@
     private int offset;
     private String useraction;
     private String frecuency;
-    
-    public String getUseraction() {
-        return useraction;
-    }
-
-    public void setUseraction(String useraction) {
-        this.useraction = useraction;
-    }
-
-    public String getFrecuency() {
-        return frecuency;
-    }
-
-    public void setFrecuency(String frecuency) {
-        this.frecuency = frecuency;
-    }
-
-    public int getOffset() {
-        int value;
-        if (getCurrentPage()> (this.getPagesToShow()/2)){
-            value = (this.getCurrentPage() - (this.getPagesToShow()/2))-1;
-        }
-        else {
-            value = 0;
-        }
-        setOffset(value);
-        return offset;
-    }
+    private boolean withLogin;
```

Comment:
this fields are alreaady present in auditlogger form, so we should abstract them to FilterAuditLoggerForm so we dont have dupe code